### DISCUSSION TITLE
Suggestion: On screen volume level indicator.

### DISCUSSION THREAD

**Discussion by auyx** [rg35xx-cfw/Koriki#60] [Ideas]
Would it be possible to get a universal on screen volume indicator when changeing the vuloume up and down? I noticed Garlic and Koriki both are missing a volume indicator so I assume it might be hard to impliment but would really improve useability.

**Comment by acmeplus** [comment]
It should be possible as an enhancement for the menu itself. Not easy to do once retroarch is running, but you can enable retroarch's volume indicator there.
